src/RestWrite.js

```diff
@@ -399,8 +399,39 @@ RestWrite.prototype.transformUser = function() {
             });
         }
       }
+      // check whether password is repeating from specified history
```

Comment:
@bhaskaryasa 
I think this entire `transformUser` method should be refactored (even before you changes to it). I think we should use this opportunity to clean it up. WDYT?